packages/design-picker/src/components/theme-style-variations-badges/theme-style-variations-badge.tsx

```diff
@@ -0,0 +1,48 @@
+import { createElement, useMemo } from 'react';
+import type { ThemeStyleVariation, ThemeStyleVariationSettingsColorPalette } from '../../types';
+import './style.scss';
+
+interface ThemeStyleVariationsBadgeProps {
+	tagName?: string;
+	variation?: ThemeStyleVariation;
+}
+
+const ThemeStyleVariationsBadge: React.FC< ThemeStyleVariationsBadgeProps > = ( {
+	tagName = 'button',
```

Comment:
We seem to always use `<span/>`. Why do we need to make this a prop? Could we just hardcode the `<span/>` ?

Reply:
Originally, I planned it to be a `<button>`, but we can't have `<button>` nested in a `<button>` 🥲
Removing `tagName` as we don't have the need for it now.